### ISSUE TITLE
About Data Loading

### ISSUE THREAD

**Issue by V1oletM** [JinyuCai95/EDESC-pytorch#1]
Hi,
Thanks for your code. It helps a lot but I'm confused about how to load the data from CIFAR-10 and CIFAR-100. May I have to load their x and y into .npy file? Also, is it available for other datasets?
Thanks a lot!

**Comment by JinyuCai95** [comment]
Hi,
   I think you can load arbitrary datasets from various types of files, e.g., .mat, .npz, .npy, .h5 are all acceptable, just make sure the format of dataset is the same as described in utils.py.

**Comment by V1oletM** [comment]
Is it right if I load the 2048-dimensional features extracted by ResNet-50 mentioned in the paper as x when using image datasets? And the does the parameter n_z have to be equal to the num_classes * d?
Thanks!

**Comment by JinyuCai95** [comment]
That's right, and n_z has to be equal to num_classes * d, you can refer to InitiallizeD.py.
